observability/distr_tracing/distr-tracing-tempo-installing.adoc

```diff
@@ -1,11 +1,16 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="dist-tracing-tempo-installing"]
-= Installing
+[id="distr-tracing-tempo-installing"]
+= Installing the {DTShortName}
 include::_attributes/common-attributes.adoc[]
-:context: dist-tracing-tempo-installing
+:context: distr-tracing-tempo-installing
 
 toc::[]
 
+[TIP]
+====
+For installing the deprecated {JaegerShortName}, see xref:../../observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-installing.adoc#dist-tracing-jaeger-installing[Installing] in the {JaegerShortName} documentation.
```

Comment:
```suggestion
For information about installing the deprecated {JaegerShortName}, see xref:../../observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-installing.adoc#dist-tracing-jaeger-installing[Installing] in the {JaegerShortName} documentation.
```
same note as above, consider not leading with this

Reply:
Thank you so much. Fixed.